middle_end/flambda2/to_cmm/to_cmm.ml

```diff
@@ -62,6 +62,13 @@ let flush_cmm_helpers_state () =
    *have* to be a root. *)
 
 let unit0 ~offsets ~make_symbol flambda_unit ~all_code =
+  (* If someone wants to add 32-bit support in the future there will be a
+     (merged) PR on ocaml-flambda/flambda-backend which can be used as a
+     guide. *)
```

Comment:
maybe add the number of the PR to make it easier to find ?

Reply:
Fixed